build-scripts/build_llvm.py

```diff
@@ -102,12 +102,27 @@ def build_llvm(llvm_dir, platform, backends, projects, use_clang=False, extra_fl
         "default": [],
     }
 
+    normal_backends = [s for s in backends if s != "ARC" and s != "Xtensa"]
+    experimental_backends = [s for s in backends if s == "ARC" or s == "Xtensa"]
```

Comment:
```suggestion
    experimental_backends = ["ARC", "Xtensa"]
```

Reply:
The backends lists are passed by CLI arguments, it's possible not to contain ARC or Xtensa